src/helpers/validators/shared/types/ValidationResult.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Result of a validation function.
+ * @author Austen Money
+ */
+type ValidationResult = (
+  | {
+    // if true, the validation was successful
+    isValid: true,
+    // the cleaned version of the value
+    cleanedValue: string,
```

Comment:
Is it true that the cleaned value is always a string? What about number fields, for example?

Reply:
Let's discuss perhaps generic typing this and allowing for other types of cleaned values.

Reply:
None of the four current validation functions allow for a non-string input - however, if the set of functions were to be expanded at some point, that would definitely be helpful. It's now generic-typed!

Reply:
Thank you!